<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VowelCrossfadingOptionWidget</class>
 <widget class="QWidget" name="VowelCrossfadingOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>463</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Vowel (C)V list</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Vowel V list (in [V] CV)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Long recoding oto block(like あ長) pattern
Use %a to stand for &quot;(C)V&quot; or &quot;V&quot; above.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="StringListModifyWidget" name="CVStringListModifyWidget" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="StringListModifyWidget" name="VStringListModifyWidget" native="true"/>
     </item>
     <item row="1" column="2">
      <widget class="StringListModifyWidget" name="LongRecordingPatternStringListModifyWidget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="doCVCheckBox">
     <property name="text">
      <string>Vowel crossfading for (C)V</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Preutterance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="CVpreutteranceSpinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Overlap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="CVoverlapSpinBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="overrideRadioButton">
          <property name="text">
           <string>Override original</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">CVBehavoirButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="copyRadioButton">
          <property name="text">
           <string>Copy as &quot;* (C)V&quot;</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">CVBehavoirButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="doVVOverlapIncreaseCheckBox">
     <property name="text">
      <string>Increase overlap to 1/2 for VV</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StringListModifyWidget</class>
   <extends>QWidget</extends>
   <header>utils/widgets/stringlistmodifywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>doCVCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="CVBehavoirButtonGroup"/>
 </buttongroups>
</ui>
